File: app/src/main/java/com/aserbao/aserbaosandroid/comon/base/viewHolder/TextViewHolder.java
package com.aserbao.aserbaosandroid.comon.base.viewHolder;

import android.view.View;
import android.widget.TextView;

import com.aserbao.aserbaosandroid.R;
import com.aserbao.aserbaosandroid.comon.base.beans.BaseRecyclerBean;
import com.aserbao.aserbaosandroid.comon.base.interfaces.IBaseRecyclerItemClickListener;

import butterknife.BindView;
import butterknife.ButterKnife;

public class TextViewHolder extends BaseClickViewHolder {
        @BindView(R.id.base_recycler_view_item_tv)
        public TextView mBaseRecyclerViewItemTv;

        public TextViewHolder(View itemView) {
            super(itemView);
            ButterKnife.bind(this, itemView);
        }

        public void setDataSource(BaseRecyclerBean classBean,int position, IBaseRecyclerItemClickListener mIBaseRecyclerItemClickListener){
            super.setDataSource(position,mIBaseRecyclerItemClickListener);
            int tag = classBean.getTag();
            String name = classBean.getName();
            if (tag >= 0) {
                itemView.setTag(tag);
                name = name + String.valueOf(tag);
            } else {
                name = name + String.valueOf(position);
            }
            mBaseRecyclerViewItemTv.setText(name);
        }

}

File: base/src/main/java/com/example/base/base/viewHolder/TextViewHolder.kt
package com.aserbao.aserbaosandroid.comon.base.viewHolder

import android.view.View
import android.widget.TextView
import butterknife.BindView
import butterknife.ButterKnife
import com.example.base.R
import com.example.base.base.beans.BaseRecyclerBean
import com.example.base.base.interfaces.IBaseRecyclerItemClickListener
import com.example.base.base.viewHolder.BaseClickViewHolder

class TextViewHolder(itemView: View) : BaseClickViewHolder(itemView) {
    var mBaseRecyclerViewItemTv: TextView? = null

    fun setDataSource(classBean: BaseRecyclerBean, position: Int, mIBaseRecyclerItemClickListener: IBaseRecyclerItemClickListener?) {
        super.setDataSource(position, mIBaseRecyclerItemClickListener)
        val tag = classBean.tag
        var name = classBean.name
        if (tag >= 0) {
            itemView.tag = tag
            name = name + tag.toString()
        } else {
            name = name + position.toString()
        }
        mBaseRecyclerViewItemTv!!.text = name
    }

    init {
        mBaseRecyclerViewItemTv = itemView.findViewById(R.id.base_recycler_view_item_tv)
    }
}